pkgs/by-name/gn/gnu-cim/package.nix

```diff
@@ -29,7 +29,18 @@ stdenv.mkDerivation rec {
     done
   '';
 
-  env.CFLAGS = lib.optionalString stdenv.cc.isClang "-Wno-return-type -Wno-error=implicit-function-declaration -Wno-error=implicit-int";
+  # lib.escapeShellArgs does not work
+  env.CFLAGS = lib.concatStringsSep " " (
+    [
+      "-Wno-error=implicit-function-declaration"
+      "-Wno-error=implicit-int"
+      "-Wno-error=return-mismatch"
+      "-Wno-error=incompatible-pointer-types"
+    ]
+    ++ lib.optionals stdenv.cc.isClang [
+      "-Wno-return-type"
```

Comment:
I built with clang and this wasn't necessary.

Reply:
`nix-build -E "with import ./. {}; pkgs.gnu-cim.override{ stdenv = pkgs.clangStdenv; }"`